ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host2.cfg

```diff
@@ -109,6 +109,75 @@ policy-map type pbr POLICY_DROP_THEN_NEXTHOP
 router bfd
    session stats snapshot interval dangerous 8
 !
+router bgp 65101
+   no bgp default ipv4-unicast
+   no bgp default ipv4-unicast transport ipv6
+   graceful-restart
```

Comment:
Is this command needed under host2? It's already present under host1.

Reply:
in host1 we are getting graceful-restart by bgp_defaults. So yes will update to not repeat